client/signup/controller.js

```diff
@@ -59,6 +61,7 @@ export default {
 				.then( ( { geo } ) => {
 					const countryCode = geo.data.body.country_short;
 					if ( 'gutenberg' === abtest( 'newSiteGutenbergOnboarding', countryCode ) ) {
+						recordTracksEvent( 'calypso_newsite_init', { flow: gutenboardingFlowID } );
```

Comment:
Or (related to above) maybe this should be a call to `trackEventWithFlow()` instead?

Reply:
Oops, ignore me, I just noticed this code isn't in the gutenboarding project.

Reply:
Although maybe we could just remove the flow prop entirely here. All this event is saying is we're entering the `newsite` code, can't imagine we'd need to filter anything in Tracks based on flow here 🤷

Not a big deal though.